pyaedt/hfss.py

```diff
@@ -3022,15 +3022,25 @@ def create_impedance_between_objects(
     def create_boundary(
         self, boundary_type=BoundaryType.PerfectE, sheet_name=None, boundary_name="", is_infinite_gnd=False
     ):
-        """Create a boundary given specific inputs.
+        """Assign a boundary condition to a sheet or surface. This method is generally
+           used by other methods in the ``Hfss`` class such as :meth:``Hfss.assign_febi``
+           or :meth:``Hfss.assign_radiation_boundary_to_faces``.
```

Comment:
```suggestion
        """Assign a boundary condition to a sheet or surface. This method is generally
           used by other methods in the ``Hfss`` class such as the :meth:``Hfss.assign_febi``
           or :meth:``Hfss.assign_radiation_boundary_to_faces`` method.
```
We need a noun after the code entity according to the style guide. Is there no way to use the method's verb in the short description. It seems odd to have the method naame use "create" and then the description use "assign" instead. Could it be "Create a boundary condition on a sheet or surface."?